packages/web-components/src/components/va-file-input/va-file-input-upgrader.ts

```diff
@@ -566,15 +566,20 @@ export const fileInput = {
 
       dropTarget.addEventListener(
         'drop',
-        function handleDrop(e) {
+        function handleDrop(e) {          
           e.preventDefault(); // Prevents browser from opening file instead of adding it to input
           this.classList.remove(DRAG_CLASS);
-
+          
           // Because of the way 'drop' works differently from 'change', need to get files from the dataTransfer object
           const dt = e.dataTransfer;
           (e.target as HTMLInputElement).files = dt.files;
+          
+          // Don't allow user to drop multiple files
+          if (dt.files.length > 1) {
+            return;
+          }
```

Comment:
This can be part of a different ticket, but I think it'd probably be beneficial to add some sort of feedback messaging so users know why uploading multiple files doesn't work.